src/Shared/UnitTests/FileUtilities_Tests.cs

```diff
@@ -536,6 +532,54 @@ public void NormalizePathInvalid()
             });
         }
 
+        [Fact]
+        [PlatformSpecific(TestPlatforms.Windows)]
+        public void GetExactPathFileExists()
+        {
+            string exactPath = FileUtilities.GetTemporaryFile();
+
+            // Since rive letters are normalized to upper case, use lower case. The rest of the path isn't all upper, so use upper case there.
+            string incorrectCasing = char.ToLower(exactPath[0]) + exactPath.Substring(1).ToUpper();
+
+            Assert.True(File.Exists(exactPath));
+            Assert.NotEqual(exactPath, incorrectCasing);
```

Comment:
We've been trying to prefer `Shouldly` assertions in new test code, even if it doesn't match the file.

Reply:
Can do